src/navbar/types.ts

```diff
@@ -60,10 +68,25 @@ export type DrawerDismissalMethod =
 export interface NavbarDrawerProps extends NavbarSharedProps {
     show: boolean;
     children: JSX.Element | JSX.Element[];
-    onClose?: (() => void) | undefined;
+    onClose?: () => void | undefined;
```

Comment:
We need the parenthesis else you are saying the function returns a void or undefined